presto-docs/src/main/sphinx/release/release-0.230.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.230
+=============
+
+General Changes
+_______________
+* Fix an issue where a SQL function fails to compile when the function implementation contains constants.
+* Fix an issue where a SQL function fails to compile when some function parameters are not referenced in the function body.
+* Fix an issue where SQL functions do not to respect null-call clause during execution.
+* Fix compilation errors for expressions over types containing an extremely large number of nested types.
+* Fix a regression in lambda evaluation (Issue#13648).
+* Fix :func:`geometry_to_bing_tile` function for geometries at -180 longitude or 85.05112878 latitude.
+* Add support to list non-builtin functions in SHOW FUNCTIONS. The feature can be turned on by the configuration property ``list-non-built-in-functions``.
+* Add support for ``DROP FUNCTION``.
+* Add :func:`combinations` function, which returns n combinations of values in an array, up to ``n = 5``.
+* Add :func:`all_match()`, :func:`any_match()`, and :func:`none_match()` functions.
+* Add :func:`expand_envelope` function to return a geometry's envelope expanded by a distance.
+* Add population counts/weights to reservoir samples.
+* Improved ``PRESTO_EXTRA_CREDENTIAL`` header parsing to allow value contain multiple ``=`` characters and url-encoding characters.
+
+Hive Changes
+____________
+* Add Alluxio client jar to ``plugin/hive-hadoop2/`` (as a runtime dependency) to avoid having to copy the jar file to all Presto servers manually when connecting to Alluxio.
+* Improve ``BatchStreamReader`` performance.
+
+Raptor Changes
+______________
+* Fix an issue where organizer does not run scheduled job due to the misreading configuration property ``storage.organization-discovery-interval``.
+* Add ``table_supports_delta_delete`` table property in Raptor to allow deletions in the background.
+  ``DELETE`` queries in Raptor can now delete the data logically while relying on compactors to delete the data physically.
+* Add support for Raptor to read data from HDFS while caching the files on local disks.
+* Add new cache module to allow using local disk for caching files from remote file systems.
+
+Verifier Changes
+________________
+* Add skipped verification results to the output for queries being filtered.
```

Comment:
@caithagoras This sentence is hard to understand. Would you rephrase?

Reply:
Changed to
```
* Add support to always output verification results for queries being skipped for verification.
```